distributed/tests/test_scheduler.py

```diff
@@ -2389,6 +2389,34 @@ async def test_idle_timeout(c, s, a, b):
     pc.stop()
 
 
+@gen_cluster(client=True)
+async def test_idle_during_update_graph(c, s, a, b):
+    class UpdateGraphTrackerPlugin(SchedulerPlugin):
+        def start(self, scheduler):
+            self.scheduler = scheduler
+            self.idle_during_update_graph = None
+
+        def update_graph(self, *args, **kwargs):
+            self.idle_during_update_graph = self.scheduler.check_idle() is None
```

Comment:
The idle check should be "is not `None`" here I think 

```suggestion
            self.idle_during_update_graph = self.scheduler.check_idle() is not None
```

Reply:
🤦